include/ord/Timing.h

```diff
@@ -63,6 +63,10 @@ class Vertex;
 class Pin;
 }  // namespace sta
 
+namespace rsz {
+  class Resizer;
+}
+
```

Comment:
Why is this needed?  I see no mention of Resizer in this header.

Reply:
Fixed that!